go.mod

```diff
@@ -3,49 +3,40 @@ module github.com/kubescape/kollector
 go 1.19
 
 require (
-	github.com/armosec/armoapi-go v0.0.112
-	github.com/armosec/cluster-notifier-api-go v0.0.3
-	github.com/armosec/utils-k8s-go v0.0.12
+	github.com/armosec/armoapi-go v0.0.114
+	github.com/armosec/cluster-notifier-api-go v0.0.5
+	github.com/armosec/utils-k8s-go v0.0.13
 	github.com/gorilla/websocket v1.5.0
 	github.com/kubescape/go-logger v0.0.11
 	github.com/kubescape/k8s-interface v0.0.82
-	go.opentelemetry.io/otel v1.11.2
-	go.opentelemetry.io/otel/trace v1.11.2
-	golang.org/x/net v0.5.0
-	k8s.io/api v0.24.3
-	k8s.io/apiextensions-apiserver v0.24.2
-	k8s.io/apimachinery v0.24.3
-	k8s.io/client-go v0.24.3
+	go.opentelemetry.io/otel v1.14.0
+	go.opentelemetry.io/otel/trace v1.14.0
+	golang.org/x/net v0.7.0
+	k8s.io/api v0.26.3
```

Comment:
We should first test it on a GKE cluster before we bump to 1.26

Reply:
We'll test on your cluster after Passover. At least it works on my 1.25

Reply:
@dwertent kollector (quay.io/matthiasb_1/kollector) running on your GKE cluster
(seems to work)